site2/docs/reference-configuration.md

```diff
@@ -162,6 +162,7 @@ Pulsar brokers are responsible for handling incoming messages from producers, di
 |exposeConsumerLevelMetricsInPrometheus|Whether to enable consumer level metrics.|false|
 |jvmGCMetricsLoggerClassName|Classname of Pluggable JVM GC metrics logger that can log GC specific metrics.|N/A|
 |bindAddress| Hostname or IP address the service binds on, default is 0.0.0.0.  |0.0.0.0|
+|bindAddresses| Additional Hostname or IP addresses the service binds on: `listener_name:scheme://host:port,...`.  ||
```

Comment:
could this filed be added to both the **Broker** section and the **Standalone** section as you have updated the `broker.conf` and the `standalone.conf` files.

Reply:
I have the same questions here.